.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,7 +4,8 @@ Many thanks for your contribution, we truly appreciate it. We will appreciate it
   - You made changes related to only one bug (create separate PRs for separate problems)
   - You are ready to defend your changes (there will be a code review)
   - You don't touch what you don't understand
-  - You ran the build locally and it passed
+  - You ran the build locally and it passed (see .travis.yml for all checks performed on PRs)
+  - Your commit messages comply with our rules (see .gitlint)
```

Comment:
@llorllale, first of all, it's an amazing idea to check the commit message.
We need this tool in all our projects.

The main problem is that we need to explain which commit message is correct/wrong.
For example, I've checked the `.gitlint` file, but most of the rules for me unclear, sorry for that.
Could you please add some documentation to contribution section about activated rules?
For example, this message cause failure, this is how it can be fixed.

Reply:
@dgroup this is an initial test. I expect to have to refine rules later on.

Documentation for gitlint is available [here](https://jorisroovers.github.io/gitlint/)